### ISSUE TITLE
Color constants in Kick project's automatically built sprites.asm resource

### ISSUE THREAD

**Issue by 3vi1** [rolandshacks/vs64#80]
Am I misunderstanding these lines at the end of the built sprite resources .asm file?

```
////////////////////////////////////////////////////////////////////////////////
// Type:         Meta Data
////////////////////////////////////////////////////////////////////////////////
.const sprites_col_background = 0
.const sprites_col_multi1 = 1
.const sprites_col_multi2 = 0
.const sprites_sprite_count = 6

```
The first three values always seem to be 0, 1, 0... whereas I was expecting them to be the color values set in SpritePad.

I have changed the shared multicolors in SpritePad, deleted sprites, and added sprites. The actual sprites and sprite count updates when it rebuilds the file, but not the col_background and col_multi values.  Re-opening the file in SpritePad shows my color changes did stick.

**Comment by rolandshacks** [comment]
Thanks for the report - it's a bug in the sprite pad reader... will be fixed asap.

**Comment by rolandshacks** [comment]
Just in case you're curious how to patch quickly: this will be the fix:

[spritepad_fix.patch](https://github.com/user-attachments/files/17968658/spritepad_fix.patch)

**Comment by rolandshacks** [comment]
Fixed with v2.5.14
https://github.com/rolandshacks/vs64/releases/latest
